stories/components/todo/todo.stories.js

```diff
@@ -25,6 +25,14 @@ export const tasks = () => html`
   <mgt-todo></mgt-todo>
 `;
 
+export const tasksWithTargetId = () => html`
+  <mgt-todo target-id="AQMkADhiZTI2MGUyLTZjYmUtNDNlYS05YTJkLWNmNzA4MTIwNWI5YQAuAAAD7zBnIbVuBE6ECinrQyLxwwEAcUAkVjKRgUeDolDvaYAd0AAAAgESAAAA"></mgt-todo>
```

Comment:
When I tested in storybook anonymously this story didn't work, here's the default task list id from the mock provider data
```suggestion
  <mgt-todo target-id="AAMkAGVmMDEzMTM4LTZmYWUtNDdkNC1hMDZiLTU1OGY5OTZhYmY4OAAuAAAAAAAiQ8W967B7TKBjgx9rVEURAQAiIsqMbYjsT5e-T7KzowPTAAAAAAESAAA="></mgt-todo>
```